### ISSUE TITLE
Error in FUN(X[[i]], ...) : !anyNA(x) is not TRUE

### ISSUE THREAD

**Issue by EmilSkindersoe** [ianjonsen/foieGras#32]
I have a data frame in with multiple individuals. It looks like this:
````{r}
    id                date             lc   lon         lat
1 Iris 2017-05-07 22:26:42  G -53.1867 69.2288
2 Iris 2017-05-08 00:48:20  G -53.0257 69.2377
3 Iris 2017-05-08 07:27:27  G -52.9513 69.2776
4 Iris 2017-05-08 08:20:10  G -52.9172 69.2785
5 Iris 2017-05-08 12:07:35  G -53.0159 69.2895
6 Iris 2017-05-08 12:47:04  G -53.0486 69.2759
````

I can fit a crw to this data and it gives me an fG_ssm object. Now the problem is when I try and plot or fit the mpm the following happens:

````{r}
> plot(fit,type=2,what="fitted")
Error in FUN(X[[i]], ...) : !anyNA(x) is not TRUE
````

I have tested that this error does not occur when running the fits on the example data "sese". Also there are no NA's for me to find anywhere in the fit (particularly not under the predicted).

**Comment by ianjonsen** [comment]
A reproducible example would be helpful, especially given the error does not occur using the `sese` example data. Also, what version of foieGras are you running, 0.7-6 (CRAN) or later from GitHub?

**Comment by EmilSkindersoe** [comment]
Hello Ian,
I am using the CRAN version (0.7-6). Uploading the model is a bit troublesome, since the file is way over the 10MB limit - so I have uploaded a subset of the model [Hvalborg_model.zip](https://github.com/ianjonsen/foieGras/files/8501374/Hvalborg_model.zip). 

After further investigation it seems to stem from the crw producing NA's for the errors on longitudal/latitudal estimates in individuals with fewer than 50 observations (like the one I uploaded). It may just be best to discard these sets due to bad fitting anyway. So I guess I have found an answer myself, but thank you for helping.
